website/build.gradle

```diff
@@ -91,30 +92,38 @@ task startDockerContainer(type: Exec) {
     "${->createDockerContainer.containerId()}" // Lazily evaluate containerId.
 }
 
+task initGitSubmodules(type: Exec) {
+    commandLine 'docker', 'exec', '-u', 'root',
+                "${->startDockerContainer.containerId()}", 'git',  'submodule', 'update', '--init',  '--recursive'
+}
+
+task installDependencies(type: Exec) {
+    commandLine 'docker', 'exec', '-u', 'root', '--workdir', "$dockerSourceDir",
+                "${->startDockerContainer.containerId()}", 'yarn', 'install'
+}
+
+task buildGithubSamples(type: Exec) {
+  commandLine 'docker', 'exec', '-u', 'root', '--workdir', "$dockerSourceDir",
+              "${->startDockerContainer.containerId()}", 'yarn', 'build_github_samples'
+}
```

Comment:
Can you try running these as non-root? The fact that docker is running as root is causing troubles for other tests, it seems.

Reply:
I wonder if we should remove the `-u root` from the other docker exec command below as well? I don't know why it's there, and I had to remove all 4 in order to be able to run `:websitePrecommit` locally.

Also once I did get it running I ran into the following error:
```
Error: Error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): SCSS processing failed: file "/opt/website/www/site/assets/scss/_table-wrapper.sass", line 12, col 35: Invalid CSS after "...the License. */": expected 1 selector or at-rule, was "*/ {}" 
```

Which seems to be another issue. Need to change the comment syntax in main.scss

Reply:
Confirmed that removing these lines seems to have fixed the issue with creating files owned by root. I ran
`find /home/jenkins/jenkins-slave/workspace/beam_PreCommit_Website_Stage_GCS_Commit/ -group root` and `find /home/jenkins/jenkins-slave/workspace/beam_PreCommit_Website_Commit/ -group root` on the workers it used and they don't have any. 

Also ran find with `-exec rm` on every worker to remove some root-owned files that I seemed to have missed before. We _should_ be good now.